components/automate-deployment/pkg/server/backup.go

```diff
@@ -39,9 +40,30 @@ func (s *server) CreateBackup(ctx context.Context, req *api.CreateBackupRequest)
 		return nil, err
 	}
 
-	sender := s.newEventSender()
+	// Tell the converger we are going to start a backup
+	t, err := converge.NewTask()
+	if err != nil {
+		s.deployment.Unlock()
+		logrus.WithError(err).Error("Failed to create task")
+		return nil, err
+	}
+
+	err = s.converger.StartBackup(t)
+	if err != nil {
+		s.deployment.Unlock()
+		logrus.WithError(err).Error("Failed to start backup")
+		return nil, err
+	}
+
+	err = <-t.C
```

Comment:
I'm not sure I'm following what's happening here. Why is a message expected on t.C?

Reply:
nvm, i think i understand.. We call sendMessage, which doesn't wait for a response.
If there aren't any use cases for not waiting for the response, maybe the function should wait.
It would also get rid of the need for creating a task.

Also, `t.Wait()` might be clearer